cmd/agentctl/commands/model.go

```diff
@@ -168,8 +167,7 @@ func newModelInspectCommand(cli agentcli.Cli) *cobra.Command {
 			return runModelInspect(cli, opts)
 		},
 	}
-	// TODO: add support for custom formatting instead of json
-	//cmd.Flags().StringVar(&opts.Format, "format", "", "Format for the output")
+	cmd.Flags().StringVarP(&opts.Format, "format", "f", "", "Format for the output")
```

Comment:
why not use "json" as default value here (instead of empty string)?

Reply:
To make the default output type configurable globally in the future for all commands. Config for agentctl might define default as `yaml` then empty string will default to YAML format instead of JSON.